lesson_10/libraries/src/loaders/kimberlee_haldane_loader.ts

```diff
@@ -0,0 +1,50 @@
+import csv from 'csv-parser';
+import fs from 'fs';
+import { Credit, MediaItem } from '../models/index.js';
+import { Loader } from './loader.js';
+
+export class KimberleeHaldaneLoader implements Loader {
+  getLoaderName(): string {
+    return 'kimberleehaldane';
+  }
+
+  async loadData(): Promise<MediaItem[]> {
+    const credits = await this.loadCredits();
+    const mediaItems = await this.loadMediaItems();
+
+    console.log(
+      `Loaded ${credits.length} credits and ${mediaItems.length} media items`,
+    );
+
+    return [...mediaItems.values()];
+  }
+
+  async loadMediaItems(): Promise<MediaItem[]> {
+    const mediaItems = [];
+    const readable = fs
+      .createReadStream('data/media_items.csv', 'utf-8')
+      .pipe(csv());
+
+    fs.createReadStream('data/media_items.csv', 'utf-8').pipe(csv());
+    for await (const row of readable) {
+      const { id, type, title, year } = row;
+      mediaItems.push(new MediaItem(id, title, type, year, []));
+      //Xavier helped me with line 31. I couldn't figure out why
```

Comment:
Move this comment to line 31 above the invocation. Also, add a space after the `//`.